share/ipld/get.go

```diff
@@ -42,6 +42,13 @@ var NumConcurrentSquares = 8
 //		so that workers spawned between each reconstruction for every new block are reused.
 var pool = workerpool.New(NumWorkersLimit)
 
+// LeavesWithProofs contains data with corresponding Merkle Proof
+type LeavesWithProofs struct {
```

Comment:
can you doc that the proof fields are empty if the user didn't request proofs?

Reply:
doced